components/scream/src/scream_config.h.in

```diff
@@ -19,8 +19,8 @@
 // How many levels to use for the vertical grid
 #define SCREAM_NUM_VERTICAL_LEV ${SCREAM_NUM_VERTICAL_LEV}
 
-// Whether this is a CUDA build
-#cmakedefine CUDA_BUILD
+// Whether this is a CUDA/HIP build
+#cmakedefine HOMMEXX_ENABLE_GPU
```

Comment:
Is this picked from hommexx_config instead?

Reply:
Yeah, I don't think we should inject HOMMEXX macros in this file. However, you are using it across the board in scream, so you can't use Hommexx_config.hpp, since that file might not be included when, e.g., we unit test physics, or the driver (where we don't link against dyn).

Imho, we should have this macro in EKAT. But for now, I would simply change the name of this macro to `SCREAM_ENABLE_GPU`, to avoid clashing against Homme's macro.